src/tools/papi-connector/README.md

```diff
@@ -1,10 +1,40 @@
-### Build papi connector for kokkos
+## Using the PAPI Connector
 
+To use the PAPI connector you have to compile it against the latest PAPI version that includes the new high-level API.
+
+#### Install PAPI (from version 6.0) or latest git version
+
+```console
+git clone https://bitbucket.org/icl/papi.git
+```   
```

Comment:
Please add instruction to build and install Papi (or say where installation guidelines can be found).